include/libcyphal/transport/udp/svc_rx_sessions.hpp

```diff
@@ -132,7 +145,7 @@ class SvcRxSession final : IRxSessionDelegate, public Interface_  // NOSONAR cpp
     void setTransferIdTimeout(const Duration timeout) override
     {
         const auto timeout_us = std::chrono::duration_cast<std::chrono::microseconds>(timeout);
-        if (timeout_us.count() > 0)
+        if (timeout_us.count() >= 0)
```

Comment:
The idiomatic form is actually `duration::zero()`:

```suggestion
        if (timeout >= Duration::zero())
```